Lesson_02 Home Work

DmytriiTsiupko opened the pull request:
Tasks:
#1. Create a program that will expect text input from the user and add whether the entered text is a "number" # or a "word"

#2. If the entered text is "number", the program must also indicate whether it is even or odd number

#3. If it is a "word", the program must show its length

- The program is supplemented with the function of defining additional data types

- Also, the program has a function of counting unique characters entered by the user

DmytriiTsiupko committed:
Lesson_02 Home Work

Tasks:
#1. Create a program that will expect text input from the user and add whether the entered text is a "number"
# or a "word"

#2. If the entered text is "number", the program must also indicate whether it is even or odd number

#3. If it is a "word", the program must show its length

- The program is supplemented with the function of defining additional data types

- Also, the program has a function of counting unique characters entered by the user

klymenok reviewed (changes requested):
The solution is good, but a little overcomplicated. The task doesn't require data validations and a string that contains at least one non-numeric character can be considered a word.  I would suggest simplifying the solution and remove redundant validation and transformation

DmytriiTsiupko committed:
Tasks:
#1. Create a program that will expect text input from the user and add whether the entered text is a "number" # or a "word"

#2. If the entered text is "number", the program must also indicate whether it is even or odd number

#3. If it is a "word", the program must show its length

Program has a function of counting unique characters entered by the user

klymenok reviewed (approved):
Looks good
